src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        int randomNumber = new Random().nextInt(100);
+        ColorSupplier fff = new ColorSupplier();
```

Comment:
According to our checklist, think about which variables should be local in the method and which should be class-level. It seems like `ColorSupplier` could be a class-level variable. Also, please follow the Java naming conventions for variable names. The name `fff` does not give any information about what this variable represents.